Tasks/src/Task1_NameAdress.java

```diff
@@ -0,0 +1,14 @@
+import java.util.Scanner;
+
+public class Task1_NameAdress {
+    public static void main (String [] args) {
+        String name, address;
+        System.out.println("What is your name ?");
+        Scanner scs1 = new Scanner(System.in);
```

Comment:
Please move initialization of Scanner( new Scanner(System.in)) to separate variable. And use here and below this variable

Reply:
And as suggestion rename scs1 and scs2 to more appropriate (like name address)